Fixes metadata of 20201125-elastic-optimized_best_fields

joshdevins opened the pull request:
Paper link was incorrect. It should be
https://www.elastic.co/blog/improving-search-relevance-with-data-driven-query-optimization

lintool commented:
Hi there, thanks for your submission! 

I've gone ahead and run the evaluation... here are your results:

```
Processing submission 20200907-h2oloo-2
Submission directory ./submissions/20200907-h2oloo-2
Verified: submission directory exists!
Verified: expected files appear in the submission directory!
Proceeding to evaluate: 

Dev run MRR@100: 0.4111622887291392
Eval run MRR@100: 0.36234372045967506

```

joshdevins committed:
Fixes metadata of 20201125-elastic-optimized_best_fields

Paper link was incorrect. It should be
https://www.elastic.co/blog/improving-search-relevance-with-data-driven-query-optimization
